source/server/http/admin.cc

```diff
@@ -1009,7 +1009,12 @@ Http::Code AdminImpl::handlerAdminHome(absl::string_view, Http::HeaderMap& respo
 
   // Prefix order is used during searching, but for printing do them in alpha order.
   for (const UrlHandler* handler : sortedHandlers()) {
-    const std::string& url = handler->prefix_;
+    absl::string_view path = handler->prefix_;
+    // Removing the leading slash from the link, so that the admin page can be
+    // rendered as part of another console, on a sub-path.
+    ASSERT(!path.empty());
+    ASSERT(path[0] == '/');
```

Comment:
Do we verify that where we register handlers that an empty handler is disallowed?   Basically if we attempt to register "" will Envoy fail at start-up or crash when the URL is hit?  A regression test would be great if we don't have one.

Reply:
Added comment in .h and ASSERTs.